client/lib/olark/index.js

```diff
@@ -148,10 +156,32 @@ const olark = {
 		olarkEvents.on( 'api.chat.onOperatorsAway', olarkActions.setOperatorsAway );
 		olarkEvents.on( 'api.chat.onOperatorsAvailable', olarkActions.setOperatorsAvailable );
 
+		olarkExpandedEvents.forEach( ( event ) => olarkEvents.on( event, () => {
```

Comment:
`event` seems to communicate to me that I should expect the event properties associated with these events, so it's a bit surprising to realize that these are actually the names of the events.

it would be clearer to me with something like:

```js
olarkExpandedEvents.forEach( eventName => olarkEvents.on( eventName, … ) );
```